packages/fluentui/docs/src/components/ComponentDoc/PerfChart/ResourcesChartTooltip.tsx

```diff
@@ -0,0 +1,52 @@
+import * as _ from 'lodash';
+import * as React from 'react';
+import { Crosshair } from 'react-vis';
+import { PerfSample } from './PerfDataContext';
+
+const values = [
```

Comment:
Under `ResourcesChartProps` we have `filter`, should extract that type and link it here:

```suggestion
const values: FlamegrillMetrics = [
```

??